### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by ai-vaibhavv** [community/community#28299] [GitHub Education]
Github student developer pack is asking to click picture but it is not recognising as the picture is of very low quality, I can upload the certificate but there is no option to.

**Comment by mizzleinetimi** [comment]
same problem here, does anyone have an answer?

**Comment by mtahir32** [comment]
I have the same problem, too. I have a legitimate proof  but my application is not proceeded.

**Comment by yezhizi** [comment]
same problem
